### DISCUSSION TITLE
Implement Xamarin.Forms.DragGestureRecognizer

### DISCUSSION THREAD

**Discussion by juliusdeblaaij** [unoplatform/uno#9492] [Ideas]
Hello,
I noticed the Xamarin.Forms.DragGestureRecognizer to be on "To be added."
With this post I would like to ask what the status of this feature is, and get a feeling for the community interest (thereby gauging a "timeline").
Kind regards,
Julius

**Comment by jeromelaban** [comment]
Thanks for the report, though this is a Xamarin.Forms feature which is not used by Uno. In which context are you seeing this?
